src/app/components/calendar/calendar.component.css

```diff
@@ -0,0 +1,23 @@
+
+.events {
+list-style: none;
+margin: 0;
+padding: 0;
+}
+
+.events .ant-badge-status {
+overflow: hidden;
+white-space: nowrap;
+width: 100%;
+text-overflow: ellipsis;
+font-size: 12px;
+}
+
+.notes-month {
+text-align: center;
+font-size: 28px;
+}
+
+.notes-month section {
+font-size: 28px;
+}
```

Comment:
you can also make this tailwind classes

Reply:
This is ng zorro stuff and they are really funny when it comes to tailwind styling